lib/ansible/plugins/cliconf/__init__.py

```diff
@@ -278,7 +278,26 @@ def get_capabilities(self):
             }
         :return: capability as json string
         """
-        pass
+        result = {}
+        result['rpc'] = self.get_base_rpc()
+        result['device_operations'] = self.get_device_operations()
+        result['network_api'] = 'cliconf'
+        return result
+
+    def get_device_operations(self):
```

Comment:
device_operation key is used in cli_config module. Setting default value here will result in false failure even if platform supports the feature for example if platform support commit and haven’t yet implemented this api in cliconf plugin will fail with error “commit is not supported on this platform”. imo it is better to fix this cli_config module. If device_operation key in not present in that case module can fail with appropriate error for.